src/Razor/src/Microsoft.CodeAnalysis.Razor/BindTagHelperDescriptorProvider.cs

```diff
@@ -388,6 +403,11 @@ private List<TagHelperDescriptor> CreateComponentBindTagHelpers(ICollection<TagH
                     builder.Metadata[BlazorMetadata.Bind.ValueAttribute] = valueAttribute.Name;
                     builder.Metadata[BlazorMetadata.Bind.ChangeAttribute] = changeAttribute.Name;
 
+                    if (expressionAttribute != null)
+                    {
+                        builder.Metadata[BlazorMetadata.Bind.ExpressionAttribute] = expressionAttribute.Name;
+                    }
+
```

Comment:
Add a unit test for this in `BindTagHelperDescriptorProviderTest` - it's OK if you want to just add another property to an existing test